apps/spruce/src/constants/routes.ts

```diff
@@ -286,11 +287,20 @@ export const getDistroSettingsRoute = (
 
 export const getWaterfallRoute = (
   projectIdentifier?: string,
-  options?: { taskFilters?: string[] },
+  options?: {
+    statusFilters?: string[];
```

Comment:
just checking; the old commits page had umbrella statuses, the new page does not. will it be ok?

Reply:
I think this ommission is ok. The goal is to primarily redirect external links and old links. The umbrella statuses always included the normal statuses as well so filters should still work the same. Besides that I don't think anyone actually filtered on status from what I've seen it was mainly task and variant filters being applied.

Reply:
you could filter out using `SortedTaskStatus` if desired to generate the most "correct" link but sounds ok